docs/releases/unreleased.md

```diff
@@ -1,7 +1,10 @@
 # Unreleased
 
-## misc
+- Introducing the `collections` module with summarization tools and data sketches working in a streaming fashion!
 
-- Move `stats.LossyCount` to `misc.LossyCount` and update its API to better match `collections.Counter`.
-- Added missing return `self` in `LossyCount`.
-- Added the Count-Min Sketch (`CountMin`) algorithm for approximate element counting.
\ No newline at end of file
+## collections
+
+- Rename `river.misc` to `river.collections`.
+- Move `stats.LossyCount` to `river.collections.HeavyHitters` and update its API to better match `collections.Counter` from the standard Python library.
+- Added missing return `self` in `HeavyHitters`.
+- Added the Count-Min Sketch (`river.collections.Counter`) algorithm for approximate element counting.
```

Comment:
After having slept on it, I think `LossyCounter` would be a good name. My issue with `Counter` is that some users will think it's the same as that of the standard library.

Reply:
I think we have a small problem:

- Lossy Count is a heavy hitters trackers (currently `HeavyHitters`)
- Count-min is for counting (currently `Counter`)

If we rename Counter to LossyCounter, users might think that Count-min is actually Lossy Count 😅

To be fair, I'm still not totally happy with `collections`. I think it is indeed misleading, although straightforward. 

`sketch` or `containers` (or another thing) could give the idea of approximation, summarization. 

Any suggestions?

Reply:
`summary`?

Reply:
Ok maybe `sketch` is a good idea then. We can say it contains lossy containers/collections

Reply:
And then yeah we call CountMin `Counter`